tests.md

```diff
@@ -0,0 +1,228 @@
+    "mod_xml": false,
+    "mysqli": "7.4.5",
+    "imagick": false,
+    "pcre": "7.4.5",
+    "xml": "7.4.5",
+    "xmlreader": "7.4.5",
+    "zlib": "7.4.5"
+  },
+  "system_utils": {
+    "curl": "7.58.0 (x86_64-pc-linux-gnu) libcurl\/7.58.0 OpenSSL\/1.1.1g zlib\/1.2.11 libidn2\/2.3.0 libpsl\/0.19.1 (+libidn2\/2.0.4) nghttp2\/1.30.0 librtmp\/2.3",
+    "ghostscript": "",
+    "imagemagick": false,
+    "openssl": "1.1.1g 21 Apr 2020"
+  },
+  "mysql_version": "mysql Ver 15.1 Distrib 10.4.12-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2",
+  "os_name": "Linux",
+  "os_version": "4.15.0-20-generic"
+}
+```
+
+As you can see, among the most important elements are the extensions that are commonly used in WordPress and some utilities that are also generally useful.
+
+The other file is the one that includes all the tests that are made (more than 10,000) giving information of the time that they take to be executed, problems that have arisen...
+
+```bash
+cat /tmp/wordpress/junit.xml
+```
+
+### Cleaning up the environment for other tests
+
+Now that we have the tests working, all that remains is to delete all the files that have been created so that we can start over. To do this we will execute the following command:
+
+```bash
+cd /tmp/phpunit-test-runner/
+php cleanup.php
+```
+
+### Improving the configuration
+
+We must not forget that the aim of this tool is to verify that our environment and infrastructure is the optimal one for WordPress to work, so, following the example above, we could make several improvements such as installing the extension of bcmath, gd, libsodium, mcrypt, mod_xml and imagick or utilities such as ghostscript and imagemagick.
+
+The goal? To be error free and have the green light for the perfect configuration.
+
+## How to report: Creating your bot for WordPress.org
+
+If you want your test [results to appear on the WordPress.org page](https://make.wordpress.org/hosting/test-results/), you can create a user for that.
+
+The first thing to do is [create a user on WordPress.org](https://login.wordpress.org/register). If your company is called *ExampleHostingCompany, Inc*, for example, you can call your users something like examplehostingcmpanybot. Keep in mind that the associated email account should be checked frequently, as emails will arrive regarding the possible operation of the tests.
+
+Create [an issue on the test page](https://github.com/WordPress/phpunit-test-runner/issues/new) asking to include the bot in the results page as a "Test Reporter", indicating the email account you used with that user.
+
+Once the user has been created in the system you will be given access to a password so that you can configure the system to send the information automatically. You will have to go to Users -> Your Profile and there generate the application password. Later you can modify the environment constant with something similar to `export WPT_REPORT_API_KEY='examplehostingbot:ABCD 1234 abcd 4567 EFGH efgh'`.
```

Comment:
"Once the user has been created in the system, you'll get an invitation to join via email."
"Then, you can [log into make/hosting](https://make.wordpress.org/hosting/wp-admin) and create an Application Password in Users -> Your Profile."
"To get things reporting properly, place the username for the bot, along with the application password in the `.env` file, which will look something like this:"

Reply:
I think it'd also be good to include something in this file / document that indicates it has to be scheduled to run for each WordPress trunk / master commit, since this does not get triggered automatically.

Many hosts do this with cron.

Reply:
done both